server_admin/topics/clients/saml/proc-creating-saml-client.adoc

```diff
@@ -0,0 +1,112 @@
+
+. Select `saml` in the `Client Protocol` drop down box.
+
+. Enter the `Client SAML Endpoint` URL. This URL is where you want the {project_name} server to send SAML requests and responses. Generally, applications have one URL for processing SAML requests. Multiple URLs can be set in the `Settings` tab of the client.
+
+. Click *Save*.  This action creates the client and brings you to the `Settings` tab.
++
+.Client Settings
+image:{project_images}/client-settings-saml.png[]
++
+The following list describes each setting:
++
+`Client ID`:: The alpha-numeric ID string that is used in OIDC requests and in the {project_name} database to identify the client. This value must match the issuer value sent with AuthNRequests. {project_name} pulls the issuer from the Authn SAML request and match it to a client by this value.
+
+`Name`:: The name for the client in {project_name} UI screen. To localize
+the name, set up a replacement string value. For example, a string value such as $\{myapp}.  See the link:{developerguide_link}[{developerguide_name}] for more information.
+
+`Description`:: The description of the client.  This setting can also be localized.
+
+`Enabled`:: When set to OFF, the client cannot request authentication.
+
+`Consent Required`:: When set to ON, users see a consent page that they can use to grant access to that application.  It will also display metadata so the user knows the exact information that the client can access.
+
+`Include AuthnStatement`:: SAML login responses may specify the authentication method used (password, etc.) as well as timestamps of the login and the session expiration.
+This is enabled by default, which means that `AuthnStatement` element will be included in login responses. Setting this to off prevents clients from determining the maximum session length, which can create  client sessions that do not expire.
+
+`Sign Documents`:: When set to ON, {project_name} sign the document using the realms private key.
+
+`Optimize REDIRECT signing key lookup`:: When set to ON, the SAML protocol messages will include the {project_name} native extension. This extension contains a hint with the signing key ID. The SAML Protocol use the extension for signature validation instead of attempting to validate the signature using keys. 
+
+This option applies to REDIRECT bindings where the signature is transferred in query parameters and there this information is not in any signature information. This is contrary to POST binding messages where key ID is always included in document signature. 
+
+This option is used where IDP and SAML Protocol are provided by the {project_name} server and adapter. This option is only relevant when `Sign Documents` is set to ON.
+
+`Sign Assertions`:: The assertion is signed and embedded within the SAML XML Auth response.
+
+`Signature Algorithm`:: Algorithm selection for signing SAML documents.
+
```

Comment:
Before "This option controls..", add

Vendors may expect a different key name or no key name.